sonar-ws/src/main/java/org/sonarqube/ws/client/organization/OrganizationService.java

```diff
@@ -67,8 +68,24 @@ public UpdateWsResponse update(UpdateWsRequest request) {
 
   public void delete(@Nullable String key) {
     PostRequest post = new PostRequest(path("delete"))
-      .setParam("key", key);
+      .setParam("organization", key);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "organization" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)